src/templates/en/2019/chapters/security.html

```diff
@@ -502,13 +502,13 @@ <h3 id="content-security-policy"><a href="#content-security-policy" class="ancho
 <p>Despite the many benefits of CSP, it can be complicated to implement on websites since its very purpose is to limit what is acceptable on a page. The policy must allow all content and resources you need and can easily get large and complex. Tools like <a href="https://report-uri.com/">Report URI</a> can help you analyze and build the appropriate policy.</p>
 <p>We find that only 5.51% of desktop pages include a CSP and only 4.73% of mobile pages include a CSP, likely due to the complexity of deployment.</p>
 <h4 id="hashnonce"><a href="#hashnonce" class="anchor-link">Hash/nonce</a></h4>
-<p>A common approach to CSP is to create a whitelist of 3rd party domains that are permitted to load content, such as JavaScript, into your pages. Creating and managing these whitelists can be difficult so <a href="https://www.w3.org/TR/CSP2/#source-list-valid-hashes">hashes</a> and <a href="https://www.w3.org/TR/CSP2/#source-list-valid-nonces">nonces</a> were introduced as an alternative approach. A hash is calculated based on contents of the script so if this is published by the website operator and the script is changed, or another script is added, then it will not match the hash and will be blocked. A nonce is a one-time code (which should be changed each time the page is loaded to prevent it being guessed) which is allowed by the CSP and which the script is tagged with. You can see an example of a nonce on this page by viewing the source to see how Google Tag Manager is loaded.</p>
+<p>A common approach to CSP is to create a list of allowed 3rd party domains that are permitted to load content, such as JavaScript, into your pages. Creating and managing these lists can be difficult so <a href="https://www.w3.org/TR/CSP2/#source-list-valid-hashes">hashes</a> and <a href="https://www.w3.org/TR/CSP2/#source-list-valid-nonces">nonces</a> were introduced as an alternative approach. A hash is calculated based on contents of the script so if this is published by the website operator and the script is changed, or another script is added, then it will not match the hash and will be blocked. A nonce is a one-time code (which should be changed each time the page is loaded to prevent it being guessed) which is allowed by the CSP and which the script is tagged with. You can see an example of a nonce on this page by viewing the source to see how Google Tag Manager is loaded.</p>
```

Comment:
In CSP, it's getting fairly common to use "allowlist" instead of a whitelist. I'd slightly favor that over the general "lists" in this sentence.